Taskfile.yml

```diff
@@ -40,7 +45,7 @@ tasks:
 
   clean-package:
     cmds:
-      - "rm -rf '{{.PACKAGE_BUILD_DIR}}'"
+      - "rm -rf '{{.G_PACKAGE_BUILD_DIR}}'"
 
   package-tar:
     deps:
```

Comment:
Shall we enforce some ordering of the task attributes in every task? e.g.
* internal
* label
* deps
* dir
* vars
* cmds
* sources
* generates
* status

Reply:
It might make more sense to put `dir` before `deps` because `deps` can occasionally occupy multiple lines.

Reply:
Yeah, I was considering that too. I think we can decide on it and do it in the next restructuring PR, so that the diff of this PR doesn't get too complex. What do you think?

Reply:
Sounds good to be done in another PR.